PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -1,27 +1,29 @@
 ﻿namespace PrincessBrideTrivia;
-
+//Timothy Nelson & Nathan Marsee, CSCD 371, Assignment 1
 public class Program
 {
+    public static int hints = 0;
     public static void Main(string[] args)
     {
         string filePath = GetFilePath();
         Question[] questions = LoadQuestions(filePath);
-
-        int numberCorrect = 0;
+        Console.WriteLine("Enter h for the answer for a hint with half credit");
+        double numberCorrect = 0.0;
         for (int i = 0; i < questions.Length; i++)
         {
             bool result = AskQuestion(questions[i]);
-            if (result)
+            if(result) 
             {
                 numberCorrect++;
             }
         }
+        numberCorrect -= 0.5 * hints;
         Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
     }
 
-    public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
+    public static string GetPercentCorrect(double numberCorrectAnswers, int numberOfQuestions)
     {
-        return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+        return Math.Round((numberCorrectAnswers / numberOfQuestions * 100),2) + "%";
```

Comment:
You are on the right track with the first change to the Program.cs file. The original code was doing integer math so with division such as 1/2 you would receive 0 ( 1/2 = 0 ). 

```suggestion
        return ( (double) numberCorrectAnswers /  (double) numberOfQuestions * 100) + "%";
```

By casting numberCorrectAnswers and nubmerOfQuestions to double, now you are able to perform correct divisions such as 1/2 = 0.5
This should resolve issues with failing tests.